<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Harison" failures="4" tests="6" name="tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest" time="0.112" errors="2" timestamp="29 May 2022 15:16:05 GMT" skipped="0">
  <testcase classname="tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest" name="testFindAll" time="0.020">
    <failure message="Collections not equal: expected: null and actual: [tokyo.huyhieu.cukcuk.model.OrderDetail@366c4480, tokyo.huyhieu.cukcuk.model.OrderDetail@2c7b5824, tokyo.huyhieu.cukcuk.model.OrderDetail@302a07d, tokyo.huyhieu.cukcuk.model.OrderDetail@5cdd09b1, tokyo.huyhieu.cukcuk.model.OrderDetail@8c11eee, tokyo.huyhieu.cukcuk.model.OrderDetail@7e8dcdaa, tokyo.huyhieu.cukcuk.model.OrderDetail@681a8b4e, tokyo.huyhieu.cukcuk.model.OrderDetail@5cbe877d, tokyo.huyhieu.cukcuk.model.OrderDetail@5c08c46a, tokyo.huyhieu.cukcuk.model.OrderDetail@4cf8b2dc, tokyo.huyhieu.cukcuk.model.OrderDetail@708400f6, tokyo.huyhieu.cukcuk.model.OrderDetail@5829e4f4, tokyo.huyhieu.cukcuk.model.OrderDetail@4218500f, tokyo.huyhieu.cukcuk.model.OrderDetail@4bff64c2, tokyo.huyhieu.cukcuk.model.OrderDetail@1b2c4efb, tokyo.huyhieu.cukcuk.model.OrderDetail@c35172e, tokyo.huyhieu.cukcuk.model.OrderDetail@c2db68f, tokyo.huyhieu.cukcuk.model.OrderDetail@3cc41abc, tokyo.huyhieu.cukcuk.model.OrderDetail@4566d049, tokyo.huyhieu.cukcuk.model.OrderDetail@61ce23ac, tokyo.huyhieu.cukcuk.model.OrderDetail@3668d4, tokyo.huyhieu.cukcuk.model.OrderDetail@1c3b9394, tokyo.huyhieu.cukcuk.model.OrderDetail@6f2cfcc2, tokyo.huyhieu.cukcuk.model.OrderDetail@7f6f61c8, tokyo.huyhieu.cukcuk.model.OrderDetail@4c2cc639, tokyo.huyhieu.cukcuk.model.OrderDetail@ccb4b1b, tokyo.huyhieu.cukcuk.model.OrderDetail@4097cac, tokyo.huyhieu.cukcuk.model.OrderDetail@ec2cc4, tokyo.huyhieu.cukcuk.model.OrderDetail@2a5b3fee, tokyo.huyhieu.cukcuk.model.OrderDetail@7c1e2a2d, tokyo.huyhieu.cukcuk.model.OrderDetail@333dd51e, tokyo.huyhieu.cukcuk.model.OrderDetail@52d645b1, tokyo.huyhieu.cukcuk.model.OrderDetail@2101b44a, tokyo.huyhieu.cukcuk.model.OrderDetail@2cc3ad05, tokyo.huyhieu.cukcuk.model.OrderDetail@710b18a6, tokyo.huyhieu.cukcuk.model.OrderDetail@119020fb, tokyo.huyhieu.cukcuk.model.OrderDetail@3d9f6567, tokyo.huyhieu.cukcuk.model.OrderDetail@c055c54, tokyo.huyhieu.cukcuk.model.OrderDetail@25e2ab5a, tokyo.huyhieu.cukcuk.model.OrderDetail@35e5d0e5, tokyo.huyhieu.cukcuk.model.OrderDetail@73173f63, tokyo.huyhieu.cukcuk.model.OrderDetail@55562aa9, tokyo.huyhieu.cukcuk.model.OrderDetail@655ef322, tokyo.huyhieu.cukcuk.model.OrderDetail@7e276594, tokyo.huyhieu.cukcuk.model.OrderDetail@3401a114, tokyo.huyhieu.cukcuk.model.OrderDetail@5066d65f, tokyo.huyhieu.cukcuk.model.OrderDetail@4233e892, tokyo.huyhieu.cukcuk.model.OrderDetail@77d2e85, tokyo.huyhieu.cukcuk.model.OrderDetail@3ecd267f, tokyo.huyhieu.cukcuk.model.OrderDetail@58ffcbd7, tokyo.huyhieu.cukcuk.model.OrderDetail@555cf22]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Collections not equal: expected: null and actual: [tokyo.huyhieu.cukcuk.model.OrderDetail@366c4480, tokyo.huyhieu.cukcuk.model.OrderDetail@2c7b5824, tokyo.huyhieu.cukcuk.model.OrderDetail@302a07d, tokyo.huyhieu.cukcuk.model.OrderDetail@5cdd09b1, tokyo.huyhieu.cukcuk.model.OrderDetail@8c11eee, tokyo.huyhieu.cukcuk.model.OrderDetail@7e8dcdaa, tokyo.huyhieu.cukcuk.model.OrderDetail@681a8b4e, tokyo.huyhieu.cukcuk.model.OrderDetail@5cbe877d, tokyo.huyhieu.cukcuk.model.OrderDetail@5c08c46a, tokyo.huyhieu.cukcuk.model.OrderDetail@4cf8b2dc, tokyo.huyhieu.cukcuk.model.OrderDetail@708400f6, tokyo.huyhieu.cukcuk.model.OrderDetail@5829e4f4, tokyo.huyhieu.cukcuk.model.OrderDetail@4218500f, tokyo.huyhieu.cukcuk.model.OrderDetail@4bff64c2, tokyo.huyhieu.cukcuk.model.OrderDetail@1b2c4efb, tokyo.huyhieu.cukcuk.model.OrderDetail@c35172e, tokyo.huyhieu.cukcuk.model.OrderDetail@c2db68f, tokyo.huyhieu.cukcuk.model.OrderDetail@3cc41abc, tokyo.huyhieu.cukcuk.model.OrderDetail@4566d049, tokyo.huyhieu.cukcuk.model.OrderDetail@61ce23ac, tokyo.huyhieu.cukcuk.model.OrderDetail@3668d4, tokyo.huyhieu.cukcuk.model.OrderDetail@1c3b9394, tokyo.huyhieu.cukcuk.model.OrderDetail@6f2cfcc2, tokyo.huyhieu.cukcuk.model.OrderDetail@7f6f61c8, tokyo.huyhieu.cukcuk.model.OrderDetail@4c2cc639, tokyo.huyhieu.cukcuk.model.OrderDetail@ccb4b1b, tokyo.huyhieu.cukcuk.model.OrderDetail@4097cac, tokyo.huyhieu.cukcuk.model.OrderDetail@ec2cc4, tokyo.huyhieu.cukcuk.model.OrderDetail@2a5b3fee, tokyo.huyhieu.cukcuk.model.OrderDetail@7c1e2a2d, tokyo.huyhieu.cukcuk.model.OrderDetail@333dd51e, tokyo.huyhieu.cukcuk.model.OrderDetail@52d645b1, tokyo.huyhieu.cukcuk.model.OrderDetail@2101b44a, tokyo.huyhieu.cukcuk.model.OrderDetail@2cc3ad05, tokyo.huyhieu.cukcuk.model.OrderDetail@710b18a6, tokyo.huyhieu.cukcuk.model.OrderDetail@119020fb, tokyo.huyhieu.cukcuk.model.OrderDetail@3d9f6567, tokyo.huyhieu.cukcuk.model.OrderDetail@c055c54, tokyo.huyhieu.cukcuk.model.OrderDetail@25e2ab5a, tokyo.huyhieu.cukcuk.model.OrderDetail@35e5d0e5, tokyo.huyhieu.cukcuk.model.OrderDetail@73173f63, tokyo.huyhieu.cukcuk.model.OrderDetail@55562aa9, tokyo.huyhieu.cukcuk.model.OrderDetail@655ef322, tokyo.huyhieu.cukcuk.model.OrderDetail@7e276594, tokyo.huyhieu.cukcuk.model.OrderDetail@3401a114, tokyo.huyhieu.cukcuk.model.OrderDetail@5066d65f, tokyo.huyhieu.cukcuk.model.OrderDetail@4233e892, tokyo.huyhieu.cukcuk.model.OrderDetail@77d2e85, tokyo.huyhieu.cukcuk.model.OrderDetail@3ecd267f, tokyo.huyhieu.cukcuk.model.OrderDetail@58ffcbd7, tokyo.huyhieu.cukcuk.model.OrderDetail@555cf22]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.assertEquals(Assert.java:815)
at org.testng.Assert.assertEquals(Assert.java:796)
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest.testFindAll(OrderDetailRepositoryNGTest.java:46)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:151)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:192)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:93)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- testFindAll -->
  <testcase classname="tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest" name="testInsert" time="0.017">
    <error message="Cannot invoke &quot;tokyo.huyhieu.cukcuk.model.OrderDetail.getIdOrder()&quot; because &quot;OrderDetail&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "tokyo.huyhieu.cukcuk.model.OrderDetail.getIdOrder()" because "OrderDetail" is null
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepository.insert(OrderDetailRepository.java:149)
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest.testInsert(OrderDetailRepositoryNGTest.java:74)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:151)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:192)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:93)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </error>
  </testcase> <!-- testInsert -->
  <testcase classname="tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest" name="testEdit" time="0.018">
    <error message="Cannot invoke &quot;tokyo.huyhieu.cukcuk.model.OrderDetail.getIdOrder()&quot; because &quot;OrderDetail&quot; is null" type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException: Cannot invoke "tokyo.huyhieu.cukcuk.model.OrderDetail.getIdOrder()" because "OrderDetail" is null
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepository.edit(OrderDetailRepository.java:183)
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest.testEdit(OrderDetailRepositoryNGTest.java:83)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:151)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:192)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:93)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </error>
  </testcase> <!-- testEdit -->
  <testcase classname="tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest" name="testRemove" time="0.019">
    <failure message="The test case is a prototype." type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The test case is a prototype.
at org.testng.Assert.fail(Assert.java:96)
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest.testRemove(OrderDetailRepositoryNGTest.java:92)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:151)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:192)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:93)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- testRemove -->
  <testcase classname="tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest" name="testFindById" time="0.018">
    <failure message="expected [null] but found [tokyo.huyhieu.cukcuk.model.OrderDetail@2e61d218]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: expected [null] but found [tokyo.huyhieu.cukcuk.model.OrderDetail@2e61d218]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:442)
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest.testFindById(OrderDetailRepositoryNGTest.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:151)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:192)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:93)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- testFindById -->
  <testcase classname="tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest" name="testFindAllByOrder" time="0.020">
    <failure message="Collections not equal: expected: null and actual: []" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: Collections not equal: expected: null and actual: []
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.assertEquals(Assert.java:815)
at org.testng.Assert.assertEquals(Assert.java:796)
at tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest.testFindAllByOrder(OrderDetailRepositoryNGTest.java:56)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:151)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:192)
at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:93)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:155)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:456)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:169)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:595)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:581)
]]>
    </failure>
  </testcase> <!-- testFindAllByOrder -->
</testsuite> <!-- tokyo.huyhieu.cukcuk.repository.OrderDetailRepositoryNGTest -->
